src/service/command-executor.js

```diff
@@ -44,6 +47,36 @@ function testCopyJUnitReports(outputDir) {
   files.forEach((file) => fs.copyFileSync(path.join(sampleDir, file), path.join(outputDir, file)));
 }
 
+function checkFileExist(ksProjectPath, newTestSuitePath) {
+  const currentTestSuitePath = path.join(ksProjectPath, 'Test Suites');
```

Comment:
We don't check in Test Suites folder only.

Reply:
Updated